boards/risc-v/mpfs/icicle/scripts/Make.defs

```diff
@@ -95,4 +95,4 @@ endif
 
 # File extensions
 
-LDFLAGS += --gc-sections -melf64lriscv
+LDFLAGS += --gc-sections -m,elf64lriscv
```

Comment:
Since `risc-v` boards are not covered by the PR anymore, this change should be reverted.

Reply:
I am in progress